scripts/metadata_comparison/test/resources/comparer/performance_json1.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": "0.0.1",
+  "calls": {
+    "root_wf.sub_wf.shard_0.call_foo.shard_0": {
+      "attempt": 3,
+      "operationId": "123",
+      "overallRuntimeSeconds": 9
```

Comment:
The 0.0.1 pre-alpha WIP digester is currently calculating the difference of the `end` and `start` times for a job, so I would like to rename this field to something like `cromwellTotalTimeSeconds` to indicate these are Cromwell times and might include polling lag.

Reply:
Also a heads up these are currently coming out as floats. 🙂

Reply:
I will try to get you some samples of "real" digests shortly.